### PR TITLE
add: `minecraft-launcher-deb`

### PR DESCRIPTION
**Pull request by theblobscp**

### CODE REVIEW

**Review by Henryws — CHANGES_REQUESTED** [review]
Add breaks and gives

**Comment by theblobscp** [comment]
Changes have been made

**Comment by Henryws** [comment]
https://github.com/pacstall/pacstall-programs/blob/master/make-a-pacscript.md

Take a look at how to use the breaks field

**Review by Henryws — CHANGES_REQUESTED** [review]
Fill out breaks with the rest of the options like `${pkgname}-git ${pkgname}-app`, etc

**Review by Henryws — CHANGES_REQUESTED** [review]
Add the rest of the breaks options. You can just copy paste from the `make-a-pacscript.md`

**Comment by Henryws** [comment]
Wait I'm such a klutz, can you remove `breaks`. My bad

File: packages/minecraft-launcher-deb/minecraft-launcher-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name="minecraft-launcher-deb"
+pkgname="minecraft-launcher-deb"
```
Comment (Henryws) [inline packages/minecraft-launcher-deb/minecraft-launcher-deb.pacscript:2]:
```suggestion
pkgname="minecraft-launcher"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by theblobscp** [comment]
> Wait I'm such a klutz, can you remove `breaks`. My bad

I am very confused

**Comment by Henryws** [comment]
I forgot that debs don't need `breaks`

**Comment by Henryws** [comment]
I can fix it for you

**Comment by theblobscp** [comment]
I saw that, but didn't know what to do. I'm almost done with the change nevertheless.

**Comment by Henryws** [comment]
Actually don't remove breaks, soo sorry, my brain is just not working as well right now 😁

File: packages/minecraft-launcher-deb/minecraft-launcher-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,8 @@
+name="minecraft-launcher-deb"
+pkgname="minecraft-launcher"
+version="928"
+breaks="${pkgname} ${pkgname}-git ${pkgname}-bin"
```
Comment (Henryws) [inline packages/minecraft-launcher-deb/minecraft-launcher-deb.pacscript:4]:
```suggestion
breaks="${pkgname} ${pkgname}-git ${pkgname}-bin ${pkgname}-app"
```

**Comment by Henryws** [comment]
Also do you know if minecraft supplies a url where the version is static?

**Comment by theblobscp** [comment]
Oh no I broke the script during the merge.

**Comment by theblobscp** [comment]
Never mind vim just broke for a second

**Comment by Henryws** [comment]
If you need help you can join our discord (https://discord.gg/yzrjXJV6K8) and I can help fix this with you. Or if you use Matrix (https://matrix.to/#/#pacstall:matrix.org)

File: packages/minecraft-launcher-deb/minecraft-launcher-deb.pacscript (hunk 2 above)
Comment (theblobscp) [reply packages/minecraft-launcher-deb/minecraft-launcher-deb.pacscript]: > Also do you know if minecraft supplies a url where the version is static?

No idea

Comment (Henryws) [reply packages/minecraft-launcher-deb/minecraft-launcher-deb.pacscript]: Because that's kind of necessary for this script to work

**Comment by Henryws** [comment]
Ok, so all that's left is to get a static version URL

**Comment by wizard-28** [comment]
> Ok, so all that's left is to get a static version URL

I guess we could skip the `hash`, if it's not possible to get a static URL.

**Comment by theblobscp** [comment]
I fixed it @wizard-28

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Add yourself as the maintainer

**Comment by theblobscp** [comment]
I don't want to give out my email, is there anything else I can use?

**Comment by wizard-28** [comment]
> I don't want to give out my email, is there anything else I can use?

I guess you could give your github noreply email.



*The highlighted part in the picture is the noreply email address*

**Comment by theblobscp** [comment]
For some reason, I only just now saw this. Changes have been made.

**Review by wizard-28 — APPROVED** [review]

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Edit `packagelist` and add `minecraft-launcher-deb` there.

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**